server/services/interventionsService.ts

```diff
@@ -103,6 +104,33 @@ export default class InterventionsService {
     return sanitisedError
   }
 
+  serializeDeliusServiceUser(deliusServiceUser: DeliusServiceUser | null): ServiceUser {
+    if (!deliusServiceUser) {
+      return {} as ServiceUser
+    }
+
+    const currentDisabilities = deliusServiceUser.disabilities
+      .filter(disability => {
+        const today = new Date().toString()
+        return disability.endDate === '' || Date.parse(disability.endDate) >= Date.parse(today)
+      })
+      .map(disability => disability.disabilityType.description)
```

Comment:
Why do we want to do this formatting instead of storing it as structured data (i.e. a list) in the interventions service? It seems like we're narrowing the range of things we could do with this data later.

Reply:
At the moment, we only want to display it on the user details page, and the shape of the data might be different once we switch to using the Probation Offender Search - how would you feel about us getting this in as is and making tweaks in future when we need to use this data for other purposes (e.g. Discover?)

Reply:
Yeah, would it be much more work to just store it as a list of strings for now and then format them in a presenter? That's what the presenters are for.

I think that without seeing examples of real data, I don't know whether just sticking the disabilities from Delius together as a comma-separated list would make sense - e.g. are they all single nouns or might they be whole sentences / narrative things?

Reply:
in terms of the format of the data i have no idea what it will look like. 

in terms of what we do with it i think this is purely informational for the SP, so as long as we can display it i don't really care how it's stored. there's a bit more overhead storing lists in the db, but it's not a big deal. pre-processing the data and storing it as a simple string is my preference though.

Reply:
> I don't know whether just sticking the disabilities from Delius together as a comma-separated list would make sense - e.g. are they all single nouns or might they be whole sentences / narrative things?

Any thoughts on this @tomsmyers? The point being that keeping the data structured gives us the flexibility to stick them into an `<ul>` or whatever later on.

Reply:
i'll make the backend changes now accept a json list for this field. sounds like it makes everything more flexible.

Reply:
I've changed this to an array of strings, I hope that's not going to be too much work to change @tomsmyers ? I'll format it nicely in the presenter when we display the page.